### PR TITLE
Lu 3239 graph ql contributor searching

### PR DESCRIPTION
**Pull request by mattprice-28**
Added api to call graphql for contributor search

### CODE REVIEW

File: params.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+postgres_connection_string: ['--connection', 'postgres://takeonadmin:aVerySecurePassword123!@validationdb.cyjaepzpx1tk.eu-west-2.rds.amazonaws.com/validationdb', '--schema', 'dev01', '-j', '-a', '--watch']
```
Comment (dllmorgan) [inline params.yml]: This shouldn't be here!

Comment (mattprice-28) [reply params.yml]: I definitely added it to the git ignore, don't know how it made it in

Comment (dllmorgan) [reply params.yml]: It's definitely in the ignore, I'm not sure how this snuck through. It's crept into the other branch too.

Comment (RyanKilbane) [reply params.yml]: To ignore everything on the path <whatever>/params.yml you need **/params.yml, not */params.yml

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -6,4 +6,5 @@ ribbon.ReadTimeout = 60000
 ribbon.ConnectTimeout = 60000
 spring.main.allow-bean-definition-overriding=true
 ribbon.eureka.enabled=false
-eureka.client.serviceUrl.defaultZone = http://eurekaserver:8761/eureka
\ No newline at end of file
+#eureka.client.serviceUrl.defaultZone = http://eurekaserver:8761/eureka
```
Comment (RyanAldy) [inline src/main/resources/application.properties]: Shall we just remove this rather than comment it out?

File: .gitignore [hunk 3]
```diff
@@ -17,10 +17,14 @@
...
+
 ### NetBeans ###
 /nbproject/private/
 /build/
 /nbbuild/
 /dist/
 /nbdist/
 /.nb-gradle/
+
+*/params.yml
```
Comment (RyanAldy) [inline .gitignore]: Should be **/params.yaml to ignore this file

**Pull request merged**